pkg/minikube/cluster/cluster.go

```diff
@@ -54,7 +56,8 @@ var (
 )
 
 const (
-	fileScheme = "file"
+	fileScheme    = "file"
```

Comment:
So far we tried to stay away from changing things in this packages. It will make things a lot harder when trying to change the model on how we consume Minikube.